pkg/adapter/nimbus-kyverno/processor/kpbuilder.go

```diff
@@ -72,17 +74,127 @@ func buildKpFor(id string, np *v1alpha1.NimbusPolicy) ([]kyvernov1.Policy, error
 	var kps []kyvernov1.Policy
 	switch id {
 	case idpool.EscapeToHost:
-		kps = append(kps, escapeToHost(np, np.Spec.NimbusRules[0].Rule))
+		kps = append(kps, escapeToHost(np))
 	case idpool.CocoWorkload:
 		kpols, err := cocoRuntimeAddition(np)
 		if err != nil {
 			return kps, err
 		}
 		kps = append(kps, kpols...)
+	case idpool.VirtualPatch:
+		kpols, err := virtualPatch(np)
+		if err != nil {
+			return kps, err
+		}
+		kps = append(kps, kpols...)
+		watchCVES(np)
 	}
 	return kps, nil
 }
 
+func watchCVES(np *v1alpha1.NimbusPolicy) {
+	rule := np.Spec.NimbusRules[0].Rule
+	schedule := "0 0 * * *"
+	if rule.Params["schedule"] != nil {
+		schedule = rule.Params["schedule"][0]
+	}
+    // Schedule the deletion of the Nimbus policy
+    c := cron.New()
+    _, err := c.AddFunc(schedule, func() {
+        fmt.Println("Checking for CVE updates and updation of policies")
+        err := deleteNimbusPolicy(np)
+        if err != nil {
+            fmt.Println(err)
+        }
+    })
+    if err != nil {
+        panic(err)
+    }
+    c.Start()
+
+}
+
+
+
+func deleteNimbusPolicy(np *v1alpha1.NimbusPolicy) error {
+    nimbusPolicyGVR := schema.GroupVersionResource{Group: "intent.security.nimbus.com", Version: "v1alpha1", Resource: "nimbuspolicies"}
+	err := client.Resource(nimbusPolicyGVR).Namespace(np.Namespace).Delete(context.TODO(), np.Name,metav1.DeleteOptions{})
+	if err != nil {
+		return fmt.Errorf("failed to delete Nimbus Policy: %s", err.Error())
+	}
+	fmt.Println("Nimbus policy deleted successfully")
+    return nil
+}
+
+func escapeToHost(np *v1alpha1.NimbusPolicy) kyvernov1.Policy {
```

Comment:
Why did you add the `escapeToHost` function in this PR?

Reply:
I just shifted the code above

Reply:
There were some minor changes in the escape to host intent implementation